subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/SubscriptionService.kt

```diff
@@ -404,7 +408,8 @@ class SubscriptionService(
             SELECT subscription.id as sub_id, subscription.type as sub_type, name, description, q,
                    notif_attributes, notif_format, endpoint_uri, endpoint_accept, times_sent, endpoint_info
             FROM subscription 
-            WHERE is_active 
+            WHERE is_active
+            AND (expires_at is null OR expires_at >= now())
```

Comment:
be careful with timezones when comparing the two dates. `expires_at` is expected to be at UTC, `now()` is not always (you can use `Instant.now().atZone(ZoneOffset.UTC)` instead)